### ISSUE TITLE
Preview window not opening

### ISSUE THREAD

**Issue by raulgenially** [jjuanramos/dbt-bigquery-preview#2]
Hi, 

All succesfully installed but nothing happen:

![Compartir pantalla - 2022-01-11 12_17_52 (1)](https://user-images.githubusercontent.com/73470842/148933567-8ccd0ea2-cd64-4d59-9ad1-778759151238.gif)

**Comment by jjuanramos** [comment]
Hey Raúl, thanks for raising the issue!
You are running Ubuntu, right?

Could you auth through this command: `gcloud auth application-default login` and try again?

Apologies for this, it's still in the early stages :sweat_smile:

**Comment by jjuanramos** [comment]
Could you please upgrade to version 0.0.8 and try again?

**Comment by raulgenially** [comment]
> Hey Raúl, thanks for raising the issue! You are running Ubuntu, right?
> 
> Could you auth through this command: `gcloud auth application-default login` and try again?
> 
> Apologies for this, it's still in the early stages 😅

Do not worry Juan! Great job anyway.

I am using Windows 10 and dbt 1.0.1. 

The dbt dbt-bigquery-preview: Preview dbt results comand works but I am not viewing the previw.
I followed dbt Labs tutorial for the OAuth setup but maybe something did not work as expected here (I am the project owner on BQ) but I am not sure if I have to create any service account.

Thoughts? Thank you!

**Comment by jjuanramos** [comment]
I see. Then there must be something related to how the extension work with the Windows filesystem.

For now I am going to unpublish it, and will publish it again once it's more tested.

Will let you know on this thread when that happens. Thank you for raising the issue Raúl, have a great day!

**Comment by raulgenially** [comment]
Hi @jjuanramos that's awensome! I will try it ASAP :) Thanks for all!

**Comment by jjuanramos** [comment]
Hey Raul, were you able to give it a shot?

**Comment by raulgenially** [comment]
Hi @jjuanramos , all has been working since the Mar 28th update but recently (a few days ago) a new message appears:

![image](https://user-images.githubusercontent.com/73470842/166414620-c86d376a-9714-458a-b4f3-203d1e59f99f.png)


with this error:

Error: ENOENT: no such file or directory, stat '/c:/Users/raul/Documentsmy-folder/my-folder2/models/sources/genially/src_my_model.sql'

**Comment by jjuanramos** [comment]
Hey Raul, sorry to hear that.

Does it happen to every single one of your models? Or just that one?

Also, could you give me the actual path of that model? Just to make sure that the path dbt-bigquery-preview expects equals the actual one.

I don't think I will be able to take a look until next week, but I will put some time into it as soon as I can.

Thanks!

**Comment by jjuanramos** [comment]
Of course! We can talk through email. My email is jjuanramos91@gmail.com :)
